lib/parity/backup.rb

```diff
@@ -31,6 +31,15 @@ def restore_from_development
     end
 
     def restore_to_development
+      drop_development_database
+      pull_remote_database_to_development
+    end
+
+    def drop_development_database
+      Kernel.system("dropdb #{development_db}")
```

Comment:
Will it exist in `$PATH` for all users with `postgresql` installed, without additional configuration?